objects/Editors/contained/3d8fa9/contained/0e35be/script.lua

```diff
@@ -0,0 +1,143 @@
+local threshold = {0,0,0,0,0,0,0,0}
+local editMode = false
+elementColors = {
+    "f9d81b",
+    "dedac1",
+    "f58546",
+    "9460b3",
+    "2b71b9",
+    "6b5f5f",
+    "3db23f",
+    "d8232c",
+}
+
+function onLoad()
+    local state = self.script_state
+    if state ~= "" then
+        local data = JSON.decode(state)
+        threshold = data.threshold or threshold
+        editMode = data.editMode or editMode
+    end
+    updateButtons()
+end
+
+function updateButtons()
+    if editMode then
+        self.clearButtons()
+        local paddingX = 1.1
+        for i = 1,8 do
+            self.createButton({
+                click_function = "button"..i,
+                function_owner = self,
+                label = ">",
+                rotation = {0,90,0},
+                font_size = 500,
+                color = hexToDec(elementColors[i]),
+                font_color = {0,0,0},
+                position = {0.5+i*paddingX,0.1,1},
+                width = 400,
+                height = 500,
+            })
+            local func = function() editThreshold(i,-1) end
```

Comment:
are these values backwards? i would think > increases values?

Reply:
They're fine as is since the buttons are rotated